mindjunkies/forums/admin.py

```
@@ -1,33 +1,34 @@
-        "reply__content",
-    )
-    list_filter = ("notification_type", "created_at", "is_read")
-    raw_id_fields = ("recipient", "actor", "topic", "reply")
+    raw_id_fields = ("author", "topic")
+
+@admin.register(Reply)
+class ReplyAdmin(admin.ModelAdmin):
+    list_display = ("parent_comment", "body","author", "created")
+    search_fields = ("body", "author__username", "parent_comment__id")
+    list_filter = ("created", "parent_comment")
+    raw_id_fields = ("author", "parent_comment")    
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)